packages/contracts-bedrock/contracts/L2/ETH.sol

```diff
@@ -11,18 +11,18 @@ import { OptimismMintableERC20 } from "../universal/OptimismMintableERC20.sol";
  * @custom:legacy
  * @custom:proxied
  * @custom:predeploy 0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000
- * @title OVM_ETH
- * @notice OVM_ETH is a legacy contract that held ETH balances before the Bedrock upgrade. All
- *         ETH balances held within this contract were migrated to the state trie as part of the
+ * @title ETH
```

Comment:
Having a contract named `ETH` is confusing.
I think we should keep the `OVM` prefix, or else change it to something like `DEAD_ETH` or just `DEAD`.

Reply:
Having a contract that held ETH balances *was* confusing, lol. Can we meet in the middle and keep it called `ETH` but move it into the `legacy` folder? I'm trying to use `legacy` as the place to stuff things that people can ignore.

Reply:
I'm not sure `legacy` is exactly 'stuff you can ignore', considering it has the proxies which run the bridge/messenger.

I just really don't like `ETH`, and if people can/should ignore it, why not give it a name that makes its status as deprecated more obvious?

Maybe @tynes can break the tie?

Reply:
`LegacyERC20ETH`?

Reply:
works for me

Reply:
kk will change